<launch>
	<!-- ========================================================== -->
	<!-- Configuration of the reinfocment learning environment      -->
	<!-- ========================================================== -->

    <!--

    v#0  -

    -->

    <!-- Learning hyperparameters -->
    <param name="action_time"       type="double"       value="5"     />
    <param name="rest_time"         type="double"       value="1"     />
    <param name="epsilon"           type="double"       value="0.25"     />

   </launch>